api/src/services/mail/index.ts

```diff
@@ -63,7 +63,7 @@ export class MailService {
 		}
 
 		try {
-			await mailer.sendMail({ to, from, subject, html, text });
+			await mailer.sendMail({ to, cc, bcc from, subject, html, text });
```

Comment:
```suggestion
			await mailer.sendMail({ to, cc, bcc, from, subject, html, text });
```

Reply:
nodemailer supports way more things even (https://nodemailer.com/message/). Should we try doing something like

```
sendMail({
   ...nodeMailerOptions,
   to, cc, bcc, from, subject
});
```

so we can allow setting any of the nodemailer options?

Reply:
Oh you're right !
I edited the file so now we have all nodemailer options.